experanto/data.py

```diff
@@ -3,20 +3,36 @@
 from collections import namedtuple
 from collections.abc import Iterable
 from pathlib import Path
+
 import numpy as np
 import torch
 from torch.utils.data import Dataset
 
 from .experiment import Experiment
-from .interpolators import ImageTrial
+from .interpolators import ImageTrial, VideoTrial
 
 
 class Mouse2pChunkedDataset(Dataset):
-    def __init__(self, root_folder: str, sampling_rate: float, chunk_size: int) -> None:
+    def __init__(
+        self,
+        root_folder: str,
+        sampling_rate: float,
+        chunk_size: int,
+        rescale: bool = False,
```

Comment:
The rescaling could/should be something that the `transforms` of the dataset get_item handle, or is there a case where it needs to be done at the level of loading the single frame?

Reply:
I agree conceptually but we need to discuss it in a bigger crowd I guess because I am not sure if we agreed on how to make the transforms logic (so far I put it here as the datasets I exported are with different dimensions)